main.cpp

```diff
@@ -410,9 +424,21 @@ int main()
 	Example::main();
     MidiKeyboard keyboard;
     keyboard.printKeys();
+    //Midikeyboard project 5 part2
+    std::cout << keyboard.pitch << " is the pitch" << std::endl;
+    keyboard.mkFunc();
+    //
     Laptop myLaptop;
+    //Laptop project 5 part2
+    std::cout << myLaptop.screenResolution << " is the resolution of the laptop screen." << std::endl;
+    myLaptop.resDescription();
+    //
     myLaptop.hardDriveDetails();
     FishTank aquarium;
+    //Fishtank project 5 part2
+    std::cout << aquarium.corals << "is the number of corals in this aquarium." << std::endl;
+    aquarium.coralLighting();
+    //
```

Comment:
your member function prints out the address of `aquarium`.   
Can you make your main() function also print out the address? 
- [ ] revise